### ISSUE TITLE
Recognize extension ".F90" as free-format Fortran

### ISSUE THREAD

**Issue by smcouch** [Glavin001/atom-beautify#1036]
Very simply, can ".F90" be added as a recognized extension for free-format Fortran?

**Comment by prettydiff** [comment]
Looks like it should already be supported: https://github.com/Glavin001/atom-beautify/blob/master/src/languages/fortran.coffee

**Comment by smcouch** [comment]
In this instance, I means specifically capital "F" .F90.  It is case-sensitive.  If I add "F90" to the extensions in fortran.coffee, it works beautifully.  Pun intended.

**Comment by Glavin001** [comment]
Huh did not even think of this. Thanks for bringing it up! 
I think I would rather ensure **all** language extensions are case-insensitive. Thoughts?

**Comment by prettydiff** [comment]
I agree that this should be a blanket enhancement for all languages.  I was surprised to discover this didn't work.
